Tests/Helper/PathHelperTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Netgen\Bundle\AdminUIBundle\Tests\Helper;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\Core\Base\Exceptions\UnauthorizedException;
+use eZ\Publish\Core\Helper\TranslationHelper;
+use eZ\Publish\Core\Repository\Values\Content\Location;
+use Netgen\Bundle\AdminUIBundle\Helper\PathHelper;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Routing\RouterInterface;
+
+class PathHelperTest extends TestCase
+{
+    /**
+     * @var \Netgen\Bundle\AdminUIBundle\Helper\PathHelper
+     */
+    protected $helper;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $locationService;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $translationHelper;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $router;
+
+    /**
+     * @var \eZ\Publish\Core\Repository\Values\Content\Location
+     */
+    protected $rootLocation;
+
+    public function setUp()
+    {
+        $this->locationService = $this->createMock(LocationService::class);
```

Comment:
We can't use `::class` constants here, since we still support PHP 5.4.

Reply:
There's also a problem with `createMock` it turns out, it doesn't exist on older versions of PHPUnit, so we'll need to use `getMockBuilder` to build the mock by hand.

Reply:
Thank you for your review. I've removed `::class` constants and I replaced `createMock` method with `getMockBuilder`.